djaoapp/urls/api.py

```diff
@@ -112,5 +112,9 @@
     url_self_provider(r'^api/todos', TodosAPIView.as_view(), 'api_todos'),
     url_prefixed(r'^api/contact',
         ContactUsAPIView.as_view(), name='api_contact_us'),
+    url_prefixed(r'^api/typeahead/q',
```

Comment:
Use `url_authenticated` decorator instead of `url_prefixed`. I was thinking we could provide places suggestions on the registration page, but there is too much risk a bot would use it and we pay for it.